cli/exchange/org.go

```diff
@@ -195,15 +195,64 @@ func OrgDel(org, userPwCreds, theOrg string, force bool) {
 		cliutils.ConfirmRemove(msgPrinter.Sprintf("Warning: this will also delete all Exchange resources owned by this org (nodes, services, patterns, etc). Are you sure you want to remove user %v from the Horizon Exchange?", theOrg))
 	}
 
-	// Search exchange for org and delete it, throw error if not found.
-	httpCode := cliutils.ExchangeDelete("Exchange", cliutils.GetExchangeUrl(), "orgs/"+theOrg, cliutils.OrgAndCreds(org, userPwCreds), []int{204, 404})
+	// Search exchange for org, throw error if not found.
+	httpCode := cliutils.ExchangeGet("Exchange", cliutils.GetExchangeUrl(), "orgs/"+theOrg, cliutils.OrgAndCreds(org, userPwCreds), []int{200, 404}, nil)
 	if httpCode == 404 {
-		cliutils.Fatal(cliutils.NOT_FOUND, msgPrinter.Sprintf("org %s not found.", theOrg))
-	} else {
-		msgPrinter.Printf("Organization %v is successfully removed.", theOrg)
-		msgPrinter.Println()
+		cliutils.Fatal(cliutils.NOT_FOUND, msgPrinter.Sprintf("org '%s' not found.", theOrg))
+	}
```

Comment:
Please swap the org checking and "are you sure" confirmation part. If org is not found, no need to display confirmation.

Reply:
What about abgot check (if it's provided by -a)? Should it remain after "are you sure" confirmation?

Reply:
Yes, CheckAgbot can also be put before "are you sure".